skill-tree/src/test/java/com/RDS/skilltree/integration/EndorsementsIntegrationTests.java

```diff
@@ -490,4 +516,65 @@ public void itShouldReturn401OnEndorsementSearchWithoutCookie() {
                         equalTo(
                                 "The access token provided is expired, revoked, malformed, or invalid for other reasons."));
     }
+
+    @Test
+    @Disabled
+    @DisplayName(
+            "Return 200, when request is made using super user cookie and status is APPROVED/REJECTED")
+    public void
+            itShouldReturn200OnUpdateEndorsementStatusWithSuperUserCookieAndAcceptOrRejectEndorsementStatus() {
+        String endorsementId = createEndorsementModel(true);
+        Response response =
+                given()
+                        .cookies(RestAPIHelper.getSuperUserCookie())
+                        .queryParam("status", EndorsementStatus.APPROVED.name())
+                        .patch("/v1/endorsements/{id}", endorsementId);
+
+        response
+                .then()
+                .statusCode(200)
+                .body("data", equalTo(null))
+                .body("message", equalTo("Successfully updated endorsement status"));
+    }
+
+    @Test
+    @Disabled
+    @DisplayName(
+            "Return 401, when request is made without using super user cookie and status is APPROVED/REJECTED")
+    public void
+            itShouldReturn401OnUpdateEndorsementStatusWithOutSuperUserCookieAndAcceptOrRejectEndorsementStatus() {
+        String endorsementId = createEndorsementModel(false);
+        Response response =
+                given()
+                        .cookies(RestAPIHelper.getUserCookie())
+                        .queryParam("status", EndorsementStatus.APPROVED.name())
+                        .patch("/v1/endorsements/{id}", endorsementId);
+
+        response
+                .then()
+                .statusCode(401)
+                .body("data", equalTo(null))
+                .body("message", equalTo("Unauthorized, Access is only available to super users"));
+    }
+
+    @Test
+    @Disabled
+    @DisplayName(
+            "Return 400, when request is made with using super user cookie and status is not APPROVED/REJECTED")
+    public void
+            itShouldReturn400OnUpdateEndorsementStatusWithSuperUserCookieAndEndorsementStatusIsNotAcceptOrReject() {
+        String endorsementId = createEndorsementModel(true);
+        Response response =
+                given()
+                        .cookies(RestAPIHelper.getSuperUserCookie())
+                        .pathParam("endorsementId", endorsementId)
+                        .queryParam("status", EndorsementStatus.PENDING.name())
+                        .patch("/v1/endorsements/{id}", endorsementId);
+
+        response
+                .then()
+                .statusCode(400)
+                .body("data", equalTo(null))
+                .body("message", equalTo("Invalid endorsement status"));
+    }
```

Comment:
@Atifsid - The error message will be `Invalid endorsement status: PENDING` in this case. Use `containsString` in line 578 instead of `equalTo` OR add the full error message to compare to.

Reply:
fixed [here](https://github.com/Real-Dev-Squad/skill-tree-backend/pull/117/commits/3fec27986459c7bbb9c8b532651be564e7eb52d4)